### PR TITLE
[release/v2.24]Add changelogs for Feb 2024 patch releases

### PR DESCRIPTION
**Pull request by archups**
**What this PR does / why we need it**:
This adds changelogs for the Feb 2024 patch releases.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: docs/changelogs/CHANGELOG-2.24.md [hunk 1]
```diff
@@ -3,6 +3,25 @@
...
+### Bugfixes
+
+- **ACTION REQUIRED:** If you use `velero.restic.deploy: true`, you will see new daemonset `node-agent` running in `velero` namespace. You might need to remove existing daemonset named `restic` manually ([#12998](https://github.com/kubermatic/kubermatic/pull/12998))
+- Fix a bug where resources deployed in the user cluster namespace on the seed, for the CSI drivers, were not being removed when the CSI driver was disabled ([#13048](https://github.com/kubermatic/kubermatic/pull/13048))
+- Fix panic, if no KubeVirt DNS config was set in the datacenter ([#13028](https://github.com/kubermatic/kubermatic/pull/13028))
+
+### Chore
+
+- Update metering to v1.1.2, fixing an error when a custom CA bundle is used ([#13013](https://github.com/kubermatic/kubermatic/pull/13013))
+- Update operating-system-manager (OSM) to [v1.4.1](https://github.com/kubermatic/operating-system-manager/releases/tag/v1.4.1)  * This includes a fix for Flatcar stable channel (version 3815.2.0) failing to provision as new nodes ([#13082](https://github.com/kubermatic/kubermatic/pull/13082))
```
Comment (embik) [inline docs/changelogs/CHANGELOG-2.24.md]: Let's categorize these as "Updates" instead.

Comment (embik) [inline docs/changelogs/CHANGELOG-2.24.md]: Can you please re-format this note? Unfortunately the notes generation tool isn't very good at that.

```suggestion
- Update metering to v1.1.2, fixing an error when a custom CA bundle is used ([#13013](https://github.com/kubermatic/kubermatic/pull/13013))
- Update operating-system-manager (OSM) to [v1.4.1](https://github.com/kubermatic/operating-system-manager/releases/tag/v1.4.1)  ([#13082](https://github.com/kubermatic/kubermatic/pull/13082))
  - This includes a fix for Flatcar stable channel (version 3815.2.0) failing to provision as new nodes
```

**Review by embik — CHANGES_REQUESTED** [review]
Some minor changes required, the comments ofc apply to the items in 2.23 and 2.22 as well.

File: docs/changelogs/CHANGELOG-2.24.md [hunk 2]
```diff
@@ -3,6 +3,23 @@
 - [v2.24.0](#v2240)
 - [v2.24.1](#v2241)
 - [v2.24.2](#v2242)
+- [v2.24.3](#v2243)
+
+## [2.24.3](https://github.com/kubermatic/kubermatic/releases/tag/2.24.3)
+
+### Bugfixes
+
+- **ACTION REQUIRED:** If you use `velero.restic.deploy: true`, you will see new daemonset `node-agent` running in `velero` namespace. You might need to remove existing daemonset named `restic` manually ([#12998](https://github.com/kubermatic/kubermatic/pull/12998))
```
Comment (xrstf) [inline docs/changelogs/CHANGELOG-2.24.md]: Same here regarding mentioning that it's relating to the velero Helm chart.

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**